docs/usage.rst

```diff
@@ -85,6 +85,31 @@ The ``namelist`` item is special in that it is explicitly stored in the ``config
 fv3gfs model, individual namelists are specified for various components of the model. As an example, the
 vertical resolution can be accessed via ``config['namelist']['fv_core_nml']['npz']``.
 
+
+Specifying individual files
+---------------------------
+
+More fine-grained control of the files that are written to the run-directory is possible using the "asset"
+representation of run-directory files. An asset is a dictionary that knows about one files's source
+location/filename, target filename, target location within the run directory and whether that file is copied or linked.
+Asset dicts can be generated with the helper function ``generate_asset``. For example::
+
+    >>> generate_asset('/path/to/filedir', 'sample_file.nc', target_location='INPUT')
+    {'source_location': '/path/to/filedir',
+    'source_name': 'sample_file.nc',
+    'target_location': 'INPUT',
+    'target_name': 'sample_file.nc',
+    'copy_method': 'copy'}
```

Comment:
Should list the possible options in this doc.

Reply:
Even though yes, this is already documented in the API autodoc through the docstring.

Reply:
I made ``generate_asset()`` link directly to its definition in the API so that it's easy to find for readers. Do you think that's sufficient? I'd rather not make this usage section much longer (or have the exact same info in two places).

Reply:
Have you tested the link works? I thought it's `:py:meth:`. If it works this is fine.

Reply:
Yes. The `:py` part is assumed.